CMakeLists.txt

```diff
@@ -110,10 +112,10 @@ add_dependencies(omrglue
 	omrgc_tracegen
 )
 
-add_subdirectory(tools)
-# Yeah, its dumb doing this here. Read note in tools/CMakeLists.txt
-if(CMAKE_CROSSCOMPILING)
-	include(${OMR_TOOLS_IMPORTFILE})
+if(OMR_TOOLS_IMPORTFILE)
+	include("${OMR_TOOLS_IMPORTFILE}")
+elseif(OMR_TOOLS)
+	add_subdirectory(tools)
```

Comment:
We should probably put a final else case here to complain if it cant find hookgen & tracegen on your path

Reply:
@rwy0717 and @dnakamura is this something we are going to do in this PR?

Reply:
I'm not sure. I've thought about it, and supporting installed tools properly might be outside the scope of this PR. I might just abort instead, and leave "support installed build tools" for a later PR. @dnakamura does that sound good?

Reply:
fair enough

Reply:
I've updated the code here to error out if the tools are disabled and not imported. I'll see about using installed hookgen/toolgen binaries in a later patch.